### PR TITLE
feat: add vento support

### PR DESCRIPTION
**Pull request by wrapperup**
Adds support for the [Vento](https://vento.js.org/) template language
Grammar can be found here: https://github.com/ventojs/tree-sitter-vento

### CODE REVIEW

File: lua/nvim-treesitter/parsers.lua [hunk 1]
```diff
@@ -2184,6 +2184,16 @@ list.vala = {
   maintainers = { "@Prince781" },
 }
 
+list.vento = {
+  install_info = {
+    url = "https://github.com/ventojs/tree-sitter-vento",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
```
Comment (clason) [inline lua/nvim-treesitter/parsers.lua:2191]:
```suggestion
```
Not needed if that is the default branch. You _do_ need to add an entry to the lockfile, though, to have CI run.

**Comment by clason** [comment]
Please also add `comment` injections. The highlights are a bit barebones; is this really all there is to the syntax?

**Comment by wrapperup** [comment]
> Please also add `comment` injections. The highlights are a bit barebones; is this really all there is to the syntax?

Yep, it's a bit like EJS where it's just JavaScript inside tags, though sometimes tags can have keywords _followed_ by JavaScript code to do special transformations on it.

File: lua/nvim-treesitter/parsers.lua [hunk 2]
```diff
@@ -2184,6 +2184,15 @@ list.vala = {
   maintainers = { "@Prince781" },
 }
 
+list.vento = {
+  install_info = {
+    url = "https://github.com/ventojs/tree-sitter-vento",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+  filetype = "vto",
```
Comment (clason) [inline lua/nvim-treesitter/parsers.lua:2192]: You might want to make a PR to Vim adding this as a recognized filetype.

**Review by ObserverOfTime — APPROVED** [review]

**Pull request merged**